core/src/main/java/hudson/model/UpdateCenter.java

```diff
@@ -225,6 +235,51 @@ public UpdateCenter() {
     UpdateCenter(@Nonnull UpdateCenterConfiguration configuration) {
         configure(configuration);
     }
+    
+    /**
+     * Creates an update center.
+     * @param config Requested configuration. May be {@code null} if defaults should be used
+     * @return Created Update center. {@link UpdateCenter} by default, but may be overridden
+     * @since TODO
+     */
+    @Nonnull
+    public static UpdateCenter createUpdateCenter(@CheckForNull UpdateCenterConfiguration config) {
+        String requiredClassName = System.getProperty(UpdateCenter.class.getName()+".className", null);
+        if (requiredClassName == null) {
+            // Use the defaul Update Center
+            LOGGER.log(Level.FINE, "Using the default Update Center implementation");
+            return createDefaultUpdateCenter(config);
+        }
+        
+        LOGGER.log(Level.INFO, "Using the custom update center: {0}", requiredClassName);
+        try {
+            final Class<?> clazz = Class.forName(requiredClassName).asSubclass(UpdateCenter.class);
+            if (!UpdateCenter.class.isAssignableFrom(clazz)) {
+                LOGGER.log(Level.SEVERE, "The specified custom Update Center {0} is not an instance of {1}. Falling back to default.",
```

Comment:
No. User requested something weird in this case. I'd better stop the initialization at all, but it may be complicated due to the current Jenkins code